doc/changes/changes_3.1.0.md

```diff
@@ -12,7 +11,9 @@ t.b.d.
 
 ## Dependencies
```

Comment:
Sidenote: 
* AFAIK the aligned sections of the changes files are Bugfixes, Features, Security Issues, Documentation, Refactorings.
* `Dependencies` and `Changes` are usually not included.